Sources/Notifications/Application+NotificationCenter.swift

```diff
@@ -0,0 +1,7 @@
+import Apodini
+
+extension Application {
+    var notificationCenter: NotificationCenter {
+        .init(app: self)
```

Comment:
If I understand correctly, NotificationCenter is more transient now. Therefore, I suggest to make NotificationCenter a value type, e.g.,
```swift
public struct NotificationCenter { ... }
```
I believe this would be a little bit more efficient than initializing and discarding a class instance many times, but I am not an expert in this area 🙂

Reply:
Good idea! I changed it to a struct now. 

Also, I noticed that a warning is logged every time `fcm` from the `Application` is recreated. That's why I retrieve it from the storage instead of recreating it.

Reply:
As far as I can see the `NotificationCenter` instance is recreated every time. This is to avoid side effects? Could it also be stored in the storage or am I missing something here?

Reply:
I will also store in storage